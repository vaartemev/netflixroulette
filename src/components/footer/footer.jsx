import React from 'react';
import { Logo } from '../logo';
import './footer.scss';

export const Footer = () => {
  return (
    <div className="footer">
      <Logo text="netflixroulette" />
    </div>
  );
};
