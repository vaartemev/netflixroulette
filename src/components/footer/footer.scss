.footer {
	max-width: 1200px;
	width: 100%;
	height: 70px;
	margin: 0 auto;
	bottom: 0;
	line-height: 70px;
	padding: 0 70px;
	background-color: rgba(0, 0, 0, .9);
}