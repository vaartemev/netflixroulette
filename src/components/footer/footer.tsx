import React from 'react';
import styled from 'styled-components';
import { Logo } from '../logo';

const FooterWrapper = styled.div`
  background-color: rgba(0, 0, 0, 0.9);
  width: 100%;
  line-height: 70px;
  padding: 0 70px;
  position: absolute;
`;

export const Footer = () => {
  return (
    <FooterWrapper>
      <Logo text="netflixroulette" />
    </FooterWrapper>
  );
};
