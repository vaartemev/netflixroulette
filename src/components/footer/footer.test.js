import { shallow } from 'enzyme';
import { Footer } from './footer';

describe('<Footer />', () => {
  const tree = shallow(<Footer text="Vladimir" />);
  it('Should be rendering with Vladimir', () => {
    expect(tree).toMatchSnapshot();
  });
});
