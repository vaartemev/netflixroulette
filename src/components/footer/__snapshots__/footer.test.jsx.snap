// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Should be rendering 1`] = `ShallowWrapper {}`;
